### ISSUE TITLE
Can't get sortkey of testcases in a testplan through XML-RPC API

### ISSUE THREAD

**Issue by soul916** [kiwitcms/Kiwi#444]
### Description of problem
When use XML-RPC API create testruns, can't get sortkey of testcases from testplan

### Component (web, API, etc)
API

### Version or commit hash (if applicable)
4.2

### How often reproducible
Always

### Steps to Reproduce

1.use rpc_client.TestCase.filter({'plan': planid]}) get testcases from a testplan
2.use rpc_client.TestRun.create create a new testrun
3.use rpc_client.TestRun.add_case(testrunid, testcaseid) add testcases to the new testrun

### Actual results
The sortkey lost in all testcases of the new testrun, can't find a way to read the sortkey of testcases from a testplan.

### Expected results
Need a method read the sortkey of testcases from a testplan.

### Additional info
NA

**Comment by atodorov** [comment]
> Need a method read the sortkey of testcases from a testplan.

Why do you need this ?  What is your use-case ?

**Comment by soul916** [comment]
@atodorov Thanks for the reply.
I use kiwi for full automatic testing. When a new build available, I use XML-RPC API create test run then run test cases. Because some test cases must run in fixed sequence, I set the sort key in test plan. When I create the test run through XML-RPC I found the sort key lost and test cases return by the caseid order. I can’t get the sort key from XML-RPC that cause the auto testing always failed.
